From e396f62d74b6cdb0000660b72fba7510a93fe627 Mon Sep 17 00:00:00 2001
From: Mario Limonciello <mario.limonciello@dell.com>
Date: Thu, 13 Feb 2020 08:57:25 -0600
Subject: [PATCH] upower: Decrease minimum battery requirement to 10%

Dell's behavioral spec for this aligns the threshold for this at 10%
rather than 30%.
---
 plugins/upower/fu-plugin-upower.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/upower/fu-plugin-upower.c b/plugins/upower/fu-plugin-upower.c
index 8d8bbe43..08fd3c02 100644
--- a/plugins/upower/fu-plugin-upower.c
+++ b/plugins/upower/fu-plugin-upower.c
@@ -9,7 +9,7 @@
 #include "fu-plugin-vfuncs.h"
 #include "fu-hash.h"
 
-#define MINIMUM_BATTERY_PERCENTAGE	30
+#define MINIMUM_BATTERY_PERCENTAGE	10
 
 struct FuPluginData {
 	GDBusProxy		*upower_proxy;
-- 
2.20.1

